includes/misc-functions.php

```diff
@@ -380,6 +380,57 @@ function give_get_purchase_session() {
 }
 
 /**
+ * Build item title for PayPal.
+ *
+ * @param array $payment_data Payment Data.
+ *
+ * @since 1.8
+ *
+ * @return string
+ */
+function give_build_paypal_item_title( $payment_data ) {
```

Comment:
You didn't update the function name like I asked

Reply:
@DevinWalker Function Updated.